xdsl/ir.py

```diff
@@ -398,19 +398,18 @@ def parent_block(self) -> Block | None:
         return self.parent
 
     @property
-    def operands(self) -> FrozenList[SSAValue]:
+    def operands(self) -> tuple[SSAValue, ...]:
```

Comment:
here, is the defined type a conversion from list to tuple?

Reply:
Not sure what you mean. The change is from a `FrozenList` to an n-tuple where all the elements are SSA values